modules/01-kafka-on-openshift.md

```diff
@@ -194,5 +194,5 @@ curl -X POST http://admin:admin@$(oc get routes grafana -o jsonpath='{.status.in
 Finally, the Apache Kafka Connect dashboard.
 
 ```shell
-curl -X POST http://admin:admin@$(oc get routes grafana -o jsonpath='{.status.ingress[0].host}{"\n"}')/api/dashboards/import -d @metrics/grafana/strimzi-kafka-connect-s2i.json --header "Content-Type: application/json"
+curl -X POST http://admin:admin@$(oc get routes grafana -o jsonpath='{.status.ingress[0].host}{"\n"}')/api/dashboards/import -d @metrics/grafana/strimzi-kafka-connect.json --header "Content-Type: application/json"
```

Comment:
as said before we can have just one JSON for both as in the Strimzi project. Metrics are not related to how Kafka Connect plugins are loaded (S2I or not).

Reply:
Are you 100% sure? When testing, I had to change `...strimzi_io_kind=\"KafkaConnect\"...` in the grafana queries to make it work.